src/server/Request.js

```diff
@@ -0,0 +1,61 @@
+'use strict'
+
+require('whatwg-fetch')
+
+module.exports = class Request {
```

Comment:
this name sounds like `new Request` would be initiating a single request. maybe something along the lines of `RequestClient` would be clearer?

Reply:
yes, agreed. I'll change that

Reply:
updated